build.gradle

```diff
@@ -124,60 +96,6 @@ task testWebApiServer_1_0_2_Platinum() {
   }
 }
 
-// task for Data Dictionary 1.5
-task testDataDictionary_1_5() {
-  dependsOn jar
-  doLast {
-    javaexec {
-      main = "io.cucumber.core.cli.Main"
-      classpath = configurations.cucumberRuntime + sourceSets.main.output + sourceSets.test.output
-      systemProperties = System.getProperties()
-      args = [
-          '--strict',
-          '--plugin',
-          'pretty',
-          '--plugin',
-          'json:build/data-dictionary-1.5.json',
-          '--plugin',
-          'html:build/data-dictionary-1.5.html',
-          '--glue',
-          'org.reso.certification.stepdefs#DataDictionary',
-          'src/main/java/org/reso/certification/features/data-dictionary',
-          '--tags',
-          (systemProperties.get('cucumber.filter.tags', '').toString().trim().length() > 0
-              ? '@DD1.5 and ' + systemProperties.get('cucumber.filter.tags') : '@DD1.5')
-      ]
-    }
-  }
-}
-
-// task for Data Dictionary 1.6
-task testDataDictionary_1_6() {
-  dependsOn jar
-  doLast {
-    javaexec {
-      main = "io.cucumber.core.cli.Main"
-      classpath = configurations.cucumberRuntime + sourceSets.main.output + sourceSets.test.output
-      systemProperties = System.getProperties()
-      args = [
-          '--strict',
-          '--plugin',
-          'pretty',
-          '--plugin',
-          'json:build/data-dictionary-1.6.json',
-          '--plugin',
-          'html:build/data-dictionary-1.6.html',
-          '--glue',
-          'org.reso.certification.stepdefs#DataDictionary',
-          'src/main/java/org/reso/certification/features/data-dictionary',
-          '--tags',
-          (systemProperties.get('cucumber.filter.tags', '').toString().trim().length() > 0
-              ? '@DD1.6 and ' + systemProperties.get('cucumber.filter.tags') : '@DD1.6')
-      ]
-    }
-  }
-}
-
```

Comment:
We don't want to keep these Data Dictionary tasks?

Reply:
They will be updated with a DD 1.7 task when the dd-test-generator branch is merged. The ones that were here were for an old set of tests that have changed a bit. Good find though.